### ISSUE TITLE
Peer doesn't push to mirror yet

### ISSUE THREAD

**Issue by damies13** [damies13/mynosql#1]
Peer doesn't push to mirror yet

**Comment by damies13** [comment]
```
	def _updatepeerindex(self, peerurl, index):
```

Currently checks if remote item is not in local index and then adds it, but need to reverse the check if local item not in remote index push to remote.

**Comment by damies13** [comment]
Also

```
	def do_POST(self):
		...
				if patharr[1].lower() == "doc":

					raise Exception("POST /doc not implimented")

```

**Comment by damies13** [comment]
Merged into v0.0.4
